lib/beaker/cli.rb

```diff
@@ -27,19 +27,16 @@ def initialize
         @execute = false
         return
       end
+
       if @options[:beaker_version_print]
         @logger.notify(beaker_version_string)
         @execute = false
         return
       end
 
-      if Beaker::Shared::SubcommandsUtil::write_config?
-        @options.dump_to_file(Beaker::Shared::SubcommandsUtil::CONFIG_PATH)
-      else
-        @logger.info("Beaker!")
-        @logger.info(beaker_version_string)
-        @logger.info(@options.dump)
-      end
+      @logger.info("Beaker!")
```

Comment:
I removed the check for writing the config, since I'm not sure we want to do this anymore